### DISCUSSION TITLE
What is the best online provder for foocus performance?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2947] [Q&A] [answered]
I want to test it in vps or AWS.
I am not sure which VPS should be for it or which indexes are important for them?

Please let me know. Thank you.

**Comment by mashb1t** [comment, marked as answer]
Depends on which hardware they offer and you are looking for.

Also see
- https://github.com/lllyasviel/Fooocus/discussions/2635
- https://github.com/lllyasviel/Fooocus/discussions/2302 (created by you)
- https://github.com/lllyasviel/Fooocus/discussions/2292 (created by you)
